### PR TITLE
Add ICal calendar feed for events + Hide past events

### PR DESCRIPTION
**Pull request by maximekabel**
## Description
Add an ICal feed for events + hide past events on the events list.
Feature was requested by Tudor Popescu

Fixes #

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have added tests to cover my changes.
- [ ] All new code is fully covered by tests (see coverage report)

### CODE REVIEW

File: apps/events/views.py [hunk 1]
```diff
@@ -1,18 +1,27 @@
...
+        events_list = (
+            Event.objects.order_by("start_date_time").filter(end_date_time__gte=datetime.datetime.today()).all()
+        )
     else:
-        events_list = Event.objects.filter(member_only=False).order_by("start_date_time")
+        events_list = (
+            Event.objects.filter(member_only=False)
+            .order_by("start_date_time")
+            .filter(end_date_time__gte=datetime.datetime.today())
+        )
```
Comment (bcvandendool) [inline apps/events/views.py:24]: Do we not want previous events to be able to be seen at all? Maybe sth similar to the photos section that allows them to still be seen when going to the past? Cause only showing upcoming events at first makes sense, but i feel there is a better solution then removing them altogether.

**Review by bcvandendool — CHANGES_REQUESTED** [review]
What is the feed going to be used for btw?

File: apps/events/models.py [hunk 2]
```diff
@@ -17,3 +18,6 @@ class Event(models.Model):
 
     def __str__(self):
         return "Event: {" + self.name + "}"
+
+    def get_absolute_url(self):
+        return "https://cosmostue.nl" + reverse("cosmos_events:events-view", kwargs={"pk": self.pk})
```
Comment (dasyad00) [inline apps/events/models.py:23]: The server should be independent of the host domain. Refer to other instances of `get_absolute_url()` in the repo.

**Review by dasyad00 — CHANGES_REQUESTED** [review]
pls also fix webpage titles

**Review by dasyad00 — APPROVED** [review]

**Pull request merged**